scripts/google_cloud_storage.py

```diff
@@ -24,10 +25,20 @@ def upload_files(
             blob.metadata = {**kwargs}
             blob.upload_from_filename(filename=output_file)
 
-    def get_files(self):
+    def get_files(self, number_of_days):
```

Comment:
Might be worth considering just passing the from date reducing the logic in the method